code-style/git/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,82 @@
+### Description
+
+_[add description of work done here]_
+
+### Spec
+Designs: _[link to design if applicable]_
+
+See Story: _[/[ISSUE-NUMBER]_
+
+### Validation
+_[delete anything irrelevant to this PR]_
+
+* [ ] This PR has visual elements, so it was reviewed by a designer.
+* [ ] This PR has code changes, and our linters still pass.
+* [ ] This PR effects production code, so it was browser tested (see below).
+* [ ] This PR has new code, so new tests were added or updated, and they pass.
+* [ ] This PR has new SCSS functions, mixins, or variables, so [Sass Tests](https://seesparkbox.com/foundry/how_and_why_we_unit_test_our_sass) were added or updated, and they pass.
+* [ ] This PR has copy changes, so copy was proofread and approved.
+* [ ] The content of this PR requires documentation, so we added a detailed description of component purpose, requirements, quirks, and instructions for use by designers and developers. Along with accessibility information if pertinent.
+
+#### To Validate
+  1. Make sure all PR Checks have passed (CircleCI, Code Climate, Snyk, etc).
+  2. Pull down all related branches.
+  3. Confirm all tests pass.
+  4. Navigate to... _[continue instructions here]_  
+
+  _[For an example of good validation instructions, check out [Bryan's Bouncy Ball PR](https://github.com/sparkbox/bouncy-ball/pull/56#issue-192153701).]_
+
+---
+
+#### Browser Testing
+_[delete if irrelevant to this issue]_
+
+#### Gold Level Support
+**macOS**
+* [ ] Safari 9+ (latest subversion)
```

Comment:
Why not just say latest version?

Reply:
I pulled these from a client project where we needed to test any device that at least 1% of users used. I think this is a good baseline for an enterprise level client, and each project manager can determine actual devices during template implementation. 

This Example Template is meant to be tailored to each individual project, so no matter what devices I list here, they will very likely be changed later. 

I'm not a fan of only testing the most recent versions of browsers, because that assumes that people using your website have the money and technology to use the newest browsers. For example, an iPhone 5 cannot upgrade past Safari 10, and there are still people who work at Sparkbox that regular use 5's, not to mention people who do not work at tech companies.

Reply:
I just thought it was interesting that many items did say latest version mixed with these other vaguer, specific ones that would grow out of date with the age of this template example.